packages/manager/src/components/authentication/APIKeyCreateModal.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from "react";
+import { ActionGroup, Button, Checkbox, Modal, Form, FormGroup, TextInput } from "@patternfly/react-core";
+import * as uuid from "uuid";
+
+interface IProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+export default (props: IProps) => {
+  const { isOpen, onClose } = props;
+  const [name, setName] = useState("");
+  const [expired, setExpired] = useState("");
+  const [scopes, setScopes] = useState<string[]>([]);
+  const [apiKey, setApiKey] = useState("");
+
+  useEffect(() => {
+    setName("");
+    setExpired("");
+    setApiKey("");
+    setScopes([]);
+  }, [isOpen]);
+
+  const onSubmit = () => {
+    setApiKey(uuid.v4());
+  };
```

Comment:
@kunyan Am I right to assume that this is temporary code, to be removed when @sayak-sarkar's api-key endpoints are ready?

Reply:
This is temporary code. 
However, I'm also curious about how to use same key in different environment if the key is generated in server side ?